lib/broccoli/ember-app.js

```diff
@@ -263,20 +276,14 @@ EmberApp.prototype._initVendorFiles = function() {
     'jquery.js': this.bowerDirectory + '/jquery/dist/jquery.js',
     'handlebars.js': handlebarsVendorFiles,
     'ember.js': {
-      development: defaultDevelopmentEmber,
-      production:  this.bowerDirectory + '/ember/ember.prod.js'
+      development: developmentEmber,
+      production: productionEmber,
     },
     'ember-testing.js': [
       this.bowerDirectory + '/ember/ember-testing.js',
       { type: 'test' }
     ],
-    'app-shims.js': [
-      this.bowerDirectory + '/ember-cli-shims/app-shims.js', {
-        exports: {
-          ember: ['default']
-        }
-      }
-    ],
+    'app-shims.js': emberShims,
```

Comment:
Exports has been unused for quite some time (at least since ember-cli@1.13.0 but likely 0.2.0).